packages/client/src/generation/generateClient.ts

```diff
@@ -181,6 +181,7 @@ export async function buildClient({
 
   const pkgJson = {
     name: getUniquePackageName(datamodel),
+    type: "commonjs",
```

Comment:
```suggestion
    type: 'commonjs',
```

weird that the lint job on ci doesn't complain 🤔